src/cnc/models/cycle_stage_base.py

```diff
@@ -56,6 +56,16 @@ def copy_template_dir(self):
         src_dir = Path(__file__).parent.parent
         cwd = self.working_dir
 
+        # Directory where assets will be copied
+        assets_dest_dir = Path(self.rendered_files_path) / "assets"
```

Comment:
I think we can rm the changes to this file and move the lambda_function_payload to the flavors folder. Then we can add some logic to handle this in `AWSEnvironmentCollection#generate_tf_assets` just like we do for the frontend routing lambda.

Reply:
Commit : https://github.com/coherenceplatform/cnc/pull/90/commits/f8dad30611a167e5565f201078d538e558f7801a